draft_editor/draft_editor_open.talon

```diff
@@ -2,5 +2,5 @@ mode: user.draft_editor
 app: vscode
```

Comment:
You may consider using a tag here instead, so apps can assert the tag without branching the file

Reply:
My reasoning was that since this file is only active when you already are in draft mode. ie mode `user.draft_editor` is enabled and if you want to change to another editor you have to update the python file there was not any gain in using another tag in this file.